include/fluent-bit/flb_http_client.h

```diff
@@ -66,8 +67,6 @@ struct flb_http_response {
     int content_length;        /* Content length set by headers */
     int chunked_encoding;      /* Chunked transfer encoding ?   */
     int connection_close;      /* connection: close ?           */
-    long chunked_cur_size;
-    long chunked_exp_size;     /* expected chunked size         */
```

Comment:
these were completely unused excluding in a mocked aws test, so I cleaned this up.